maat-court-data-api/src/main/java/gov/uk/courtdata/entity/WQHearingEntity.java

```diff
@@ -36,4 +36,14 @@
     @Column(name = "UPDATED_DATE_TIME")
     @UpdateTimestamp
     private LocalDateTime updatedDateTime;
-}
+
+    @Column(name = "TX_ID")
+    private Integer txId;
+    @Column(name = "CASE_ID")
+    private Integer caseId;
+    @Column(name = "CASE_URN")
+    private String caseUrn;
+
+    @Column(name = "RESULT_CODES")
```

Comment:
How do we store these result codes?

Reply:
We will persist as a part of hearing resulted processing.